src/main/k/working/imp/merge/2.merge-output

```diff
@@ -0,0 +1,77 @@
+\rewrites{SortGeneratedTopCell{}}(
```

Comment:
This shouldn't be checked in. Could we remove it, and add a `.gitignore` rule for it?